src/object_store/Cargo.toml

```
@@ -28,9 +28,19 @@ hyper-rustls = { version = "0.24.2", features = ["webpki-roots"] }
 hyper-tls = "0.5.0"
 itertools = { workspace = true }
 madsim = "0.2.27"
-opendal = "0.45.1"
+opendal = { version = "0.46", features = [
+    "services-azblob",
+    "services-fs",
+    "services-gcs",
+    "services-memory",
+    "services-obs",
+    "services-oss",
+    "services-s3",
+    "services-webhdfs",
+    "services-azfile",
+] }
```

So after OpenDAL 0.46, if we want to enable hdfs, we need to manually add the hdfs feature. So how about add it here and comment it out?
```suggestion
opendal = { version = "0.46", features = [
    "services-azblob",
    "services-fs",
    "services-gcs",
    "services-memory",
    "services-obs",
    "services-oss",
    "services-s3",
    "services-webhdfs",
    "services-azfile",
    #  "services-hdfs",
] }
```